hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/utils/FederationStateStoreFacade.java

```diff
@@ -645,7 +675,6 @@ public boolean equals(Object obj) {
     TResult invoke(T input) throws Exception;
   }
 
-
```

Comment:
Avoid

Reply:
Thank you very much for your help reviewing the code, I will fix it.